include/gnc_constants.hpp

```diff
@@ -0,0 +1,87 @@
+//
+// include/gnc_constants.hpp
+// PSim
+//
+// Contributors:
+//   Kyle Krol  kpk63@cornell.edu
+//
+// Pathfinder for Autonomous Navigation
+// Space Systems Design Studio
+// Cornell Univeristy
+//
+
+/* - NOTICE -
+ * 
+ * Talk to Kyle Krol and/or Nathan Zimmerberg before editing this file.
+ *
+ * Values in this file should be seldomly edited and have hidden
+ * interdependancies. The interdependancies and explicit calculation of the
+ * below parameters are specified in MATLAB/config.m. */
+
+#ifndef PAN_PSIM_INCLUDE_GNC_CONSTANTS_HPP_
+#define PAN_PSIM_INCLUDE_GNC_CONSTANTS_HPP_
+
+#include "lin.hpp"
+
+namespace gnc {
+namespace constant {
+
+/** Good old pi. */
+constexpr static double pi = 3.141592653589793;
```

Comment:
I would add the `L` suffix to all the double constants, just to make sure they don't get compressed for teensy. [https://forum.pjrc.com/threads/47790-double-literals-need-to-be-marked-as-long-double](https://forum.pjrc.com/threads/47790-double-literals-need-to-be-marked-as-long-double)

Reply:
Oooh okay thanks! Will do!